<?xml version="1.0"?>
<launch>
  <param name="debug_mode" value="false" />
  <param name="sim_mode" value="true" />

  <param name="gimbal_mode" value="true" />
  <param name="gimbal_position_topic" value="/wavesim/gimbal/frame/orientation/inertial" />
  <param name="gimbal_frame_orientation_topic" value="/wavesim/gimbal/frame/orientation/inertial" />
  <param name="gimbal_joint_orientation_topic" value="/wavesim/gimbal/joint/orientation/inertial" />

  <node name="gazebo" type="run_gazebo.sh" pkg="wavesim_ros" output="screen" />
  <node name="world" type="world_node" pkg="wavesim_ros" output="screen" />
  <node name="camera" type="camera_node" pkg="wavesim_ros" output="screen" />
</launch>
